### DISCUSSION TITLE
关于skynet.response的用法

### DISCUSSION THREAD

**Discussion by ericzhanchina** [cloudwu/skynet#1748] [General]
skynet.response应该是用来延迟恢复客户端的接口，等服务器端有数据的时候再回复客户端，这样的话，会不会在gated服有很多这种挂起，因为没有agent可能都需要挂起一个任务等待合适的时机，从服务器收消息。如果有10000个agent，那就要挂起10000个任务，这样会不会造成gated的执行变慢呢？

请教，不知道我的用法对不对。

**Comment by terry8210** [comment]
skynet.response尽量不要用于客户端与服务器通讯，或者服务端不接收前端的RESPONSE处理。

**Reply by ericzhanchina** [reply]
我怎么记得看云风的文章就是拿了一个邮件红点的例子，避免客户端轮训服务器查询是否有邮件，然后使用了skynet.response
